src/domain/event/eventKeywords/EventKeywords.tsx

```diff
@@ -1,14 +1,17 @@
 import React, { ReactElement } from 'react';
 
-import { EventFieldsFragment } from '../../../generated/graphql';
+import {
+  EventFieldsFragment,
+  EventsFieldsFragment,
```

Comment:
Hmm this is the downside passing whole event object as a prop and trying to add types 🤔 Hopefully we don't need more of these field fragments in the future :)

Reply:
True. Otherwise I would have managed with 1 file change.